readme.md

```diff
@@ -84,17 +84,17 @@ git commit -m 'version:up command added'
 
 **4. Increase version when needed**
 ```bash
-yarn run version:up -- --major
+yarn version:up --major
 ```
 
 Or via npm:
 ```bash
-npm run version:up -- --major
+npm version:up --major
```

Comment:
Please, add ` --` before params for the command, as you said in the comment ;)

```
remember to put `--` before options if you are using npm, with yarn this is not needed
```

Reply:
Ups guess I was a little too hasty in replacing things, didn't notice this was the npm version